drivers/gles3/storage/material_storage.cpp

```diff
@@ -3013,19 +3013,19 @@ void SceneShaderData::set_code(const String &p_code) {
 
 #ifdef DEBUG_ENABLED
 	if (uses_particle_trails) {
-		WARN_PRINT_ONCE_ED("Particle trails are only available when using the Forward+ or Mobile rendering backends.");
+		WARN_PRINT_ONCE_ED("Particle trails are only available when using the Forward+ or Mobile renderers.");
 	}
 
 	if (uses_sss) {
-		WARN_PRINT_ONCE_ED("Sub-surface scattering is only available when using the Forward+ rendering backend.");
+		WARN_PRINT_ONCE_ED("Sub-surface scattering is only available when using the Forward+ renderer.");
```

Comment:
If it is of any importance, this is **never** called "sub-surface" in the class reference. There's a few more warnings that call it this way, so feel free to dismiss for another PR in the future.
```suggestion
		WARN_PRINT_ONCE_ED("Subsurface scattering is only available when using the Forward+ renderer.");
```

Reply:
Oh cool, one more maybe hyphenated compound word to add to https://github.com/godotengine/godot-docs/issues/10218 😄